src/main/java/com/ait/lienzo/client/core/shape/PolyLine.java

```diff
@@ -202,6 +207,68 @@ public Point2D getHeadOffsetPoint()
         return getBoundingBoxAttributesComposed(Attribute.POINTS, Attribute.CORNER_RADIUS);
     }
 
+    @Override
+    public Point2D adjustPoint(double x, double y, double deltaX, double deltaY) {
+        Point2DArray points = getPoint2DArray();
+
+        Point2D before = null;
+        Point2D target = null;
+        Point2D after = null;
+
+        for (Point2D point: points) {
+            after = point;
+
+            if (target != null) {
+                if (target.getX() == x && target.getY() == y) {
```

Comment:
you can insert the `target != null` check on the same if statement bellow.

Reply:
I agree with @tiagodolphine. There is one extra line. Target can be checked on null in one line.